### PR TITLE
Sidebar content

### PR DESCRIPTION
**Pull request by dodecadumpling**
id's 1 to 7 have photos in them.
I have also included the foundation stone image Maria forwarded.
Others id photos to follow later ....

### CODE REVIEW

File: src/data/info.json [hunk 1]
```diff
@@ -62,12 +139,24 @@
...
+    "text": "In 1927, Leslie Wilkinson revised the original campus plan and established the fundamentals of the modern campus. The plan reoriented the campus to the north – south alignment that dominates today. This spine established the three major open spaces:  The Court of Honour named by Alsop (Whitfeld Court), The Great Court, and James Oval. Dr W Somerville promoted the concept that the area of land bounded on the north by Stirling Highway, on the south by Winthrop Hall, on the east by Hackett Hall, and on the west by the Administration Building to be named “Whitfeld Court”, so named by Senate in September 1940.",
+    "images": [
```
Comment (Drewbi) [inline src/data/info.json:154]: I'm not sure about using this image here, We'll ask fran

Comment (dodecadumpling) [reply src/data/info.json:154]: If that was the picture of Whitfeld statue bust, that was a placeholder. 

I have corrected it with the latest commit. Sorry about that.

Comment (Drewbi) [reply src/data/info.json:154]: Ah good good

**Review by Drewbi — APPROVED** [review]
Great work, really brings it together having images

**Pull request merged**